docs/DeveloperGuide.adoc

```diff
@@ -1136,25 +1169,49 @@ Use case resumes at step 2.
 +
 Use case ends.
 
+*Extensions*
+* 2a. Lead entered has a phone number or email address stored earlier.
+** 2a1. CRM Book displays message that a duplicate is detected.
++
+Use case ends.
+
+* 2b. Lead entered does not have a phone number or email address stored earlier.
+** 2b1. CRM Book adds the Lead.
+** 2b2. CRM Book displays message that the Lead has been added.
++
+Use case ends.
+
+=== Use case: User requests to import a CSV file to CRM Book
+
+*MSS*
+
+1.  User enters command for importing file.
+2.  CRM opens the corresponding CSV file.
+3.  CRM processes data and lists all persons including newly added ones.
```

Comment:
@zhuleyan this does not follow the MSS principle. You need to take turns, so 3 cannot be the CRM again, it has to be the user.